build/changeSets/2019/convertUitype22and24to21.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/*************************************************************************************************
+ * Copyright 2018 JPL TSolucio, S.L. -- This file is a part of TSOLUCIO coreBOS Customizations.
+* Licensed under the vtiger CRM Public License Version 1.1 (the "License"); you may not use this
+* file except in compliance with the License. You can redistribute it and/or modify it
+* under the terms of the License. JPL TSolucio, S.L. reserves all rights not expressly
+* granted by the License. coreBOS distributed by JPL TSolucio S.L. is distributed in
+* the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+* warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. Unless required by
+* applicable law or agreed to in writing, software distributed under the License is
+* distributed on an "AS IS" BASIS, WITHOUT ANY WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied. See the License for the specific language governing
+* permissions and limitations under the License. You may obtain a copy of the License
+* at <http://corebos.org/documentation/doku.php?id=en:devel:vpl11>
+*************************************************************************************************/
+
+class convertUitype22and24to21 extends cbupdaterWorker {
+
+	public function applyChange() {
+		global $adb;
+		if ($this->hasError()) {
+			$this->sendError();
+		}
+		if ($this->isApplied()) {
+			$this->sendMsg('Changeset '.get_class($this).' already applied!');
+		} else {
+            $tabid_helpdesk = getTabid('HelpDesk');
+            $tabid_qoutes = getTabid('Quotes');
+            $tabid_purchaseorder = getTabid('PurchaseOrder');
+            $tabid_salesorder = getTabid('SalesOrder');
+            $tabid_invoice = getTabid('Invoice');
+            $this->ExecuteQuery("UPDATE vtiger_field SET uitype = 21 WHERE tabid =? and (columnname = ?)", array($tabid_helpdesk, 'title'));
+            $this->ExecuteQuery("UPDATE vtiger_field SET uitype = 21 WHERE tabid =? and (columnname = ? OR columnname = ?)", array($tabid_qoutes, 'bill_street', 'ship_street'));
+            $this->ExecuteQuery("UPDATE vtiger_field SET uitype = 21 WHERE tabid =? and (columnname = ? OR columnname = ?)", array($tabid_purchaseorder, 'bill_street', 'ship_street'));
+            $this->ExecuteQuery("UPDATE vtiger_field SET uitype = 21 WHERE tabid =? and (columnname = ? OR columnname = ?)", array($tabid_salesorder, 'bill_street', 'ship_street'));
+            $this->ExecuteQuery("UPDATE vtiger_field SET uitype = 21 WHERE tabid =? and (columnname = ? OR columnname = ?)", array($tabid_invoice, 'bill_street', 'ship_street'));
+			$this->sendMsg('Changeset '.get_class($this).' applied!');
```

Comment:
same comment as the previous changeset

Reply:
yes that too I see

Reply:
same comment as the previous changeset about web service